nhd/NHDScheduler.py

```diff
@@ -171,6 +192,19 @@ def LoadDeployedConfigs(self):
             else:
                 self.logger.info(f'Pod {p[1]}.{p[0]} is in state: {p[3]}')
 
+    def LoadNodeDeployedConfigs(self,node_name):
+        """ Loads any configs that were already deployed by this scheduler on the node "node_name"
+            This typically happens when k8s node receives UNCORDON signal from TriadController """
+        self.logger.info('Looking for any pods already deployed with used resources')
+        pods = self.k8s.GetNodeScheduledPods(sched_name=self.sched_name, node_name=node_name)
+        self.logger.info(f'Found scheduled pods: {pods} for node: {node_name}')
+        for p in pods:
+            if p[3] in ('Running', 'CrashLoopBackOff', 'Pending'):
```

Comment:
The fourth value in the tuple returned by self.k8s.GetNodeScheduledPods is the pod's status.phase.  This is checking for 'CrashLoopBackOff,' yet that is not a valid pod status.phase (I believe it is a container state).  Further, this is not reclaiming resources of pods with in the Failed state (e.g. evicted due to overrunning ephemeral storage due to runaway logs), so this would have the effect of double booking the resources consumed of the failed pod by failing to update the accounting properly.